utils/io/progressbar.go

```diff
@@ -0,0 +1,12 @@
+package io
+
+import "io"
+
+type ProgressBar interface {
```

Comment:
I suggest moving this interface into *progress.go*, together with *ProgressMgr* intefface.
Let's also rename it to *Progress* and remove the word "Bar" from its comments and methods.